source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -180,28 +171,70 @@ to the ``FindOne()`` method, which performs the following actions:
 - Skips the first two matched documents
 
 .. io-code-block::
-   :copyable: true
+  :copyable: true
 
-   .. input::
+  .. input:: /includes/fundamentals/code-snippets/CRUD/retrieve.go
+      :start-after: begin find one docs
+      :end-before: end find one docs
       :language: go
+      :dedent:
+
+  .. output::
+      :language: none
+      :visible: false
+
+      {"Item":"Masala","Rating":9,"DateOrdered":"2009-11-12T05:00:00Z"}
+
+.. _golang-find-one-by-id:
+
+Find One by ObjectId Example
+````````````````````````````
 
-      filter := bson.D{{"date_ordered", bson.D{{"$lte", time.Date(2009, 11, 30, 0, 0, 0, 0, time.Local)}}}}
-      opts := options.FindOne().SetSkip(2)
-      
-      var result Review
-      err := coll.FindOne(context.TODO(), filter, opts).Decode(&result)
-      if err != nil {
-          panic(err)
+This example defines an ``id`` variable with a value of type ``ObjectId``
+and specifies a query filter with the ``id`` variable. The value of ``id``
```

Comment:
S: avoid with where possible - [sg](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using:~:text=word%2C%20username.-,using,-by%20using%20with)
```suggestion
This example defines an ``id`` variable with a value of type ``ObjectId``
and specifies a query filter that matches documents with an ``_id`` field that matches this variable.
The value of ``id``
```

Reply:
Then, I feel like the first sentence is sufficient to delete the second sentence.

Reply:
Slightly different wording, but fixed